### ISSUE TITLE
ReEDS fails silently if path is too long

### ISSUE THREAD

**Issue by samgdotson** [NREL/ReEDS-2.0#195]
### Reproduction Notes
Platform: Windows 11
GAMS Version: 48.1.0
ReEDS Version: 2024.7.0
Conda environment: reeds2
Package manager: Mamba
Julia version: 1.11.2
System specs
- RAM: 32GB
- CPU: Intel Core Ultra 9
Solver: CPLEX

Working Directory:
`C:/Users/Sdotson/One Drive - Union of Concerned Scientists/Documents/GitHub`

Commands executed:
```bash
git clone git@github.com:nrel/ReEDS-2.0.git
cd ReEDS-2.0
git lfs install
mamba env create -f environment.yml
mamba activate reeds2
julia --project=. instantiate.jl 
```

### Description of Issue
After following the steps above, none of the test cases I ran yielded any results despite saying `Run has finished` after a fraction of a second. All of the files the documentation recommends I inspect were either empty or nonexistent. This suggests the model failed silently.

I tried running the gams model command found in the `call_case.bat` file generated at the beginning of a run. This command also failed with a `Input line too long` error message.

I moved the ReEDS repository from the working directory above, to a new working directory `C:/Users/SDotson/` and the model ran like a charm.

### Potential Solutions
The troubleshooting documentation should be updated to reflect this possible failure mode. It might also be interesting to note that the GAMS command is getting close to the maximum input line length of 8191 characters (at least in Windows).

**Comment by boxblox** [comment]
Hi Sam, hard to say exactly... but GAMS does not work well with sync'd folders like OneDrive, GoogleDrive, etc... the synch'ing software can interfere in weird ways.  However, you might be popping up against a limitation (255 chars) in GAMS that is both unfortunate and hard to bypass.  You're root path isn't long enough, but there are probably subdirectories that get appended which could push things over.  You're work around should be robust since you accomplish two things at once.  The ultimate line length for a command can be fixed by using a [parameter file](https://www.gams.com/latest/docs/UG_GamsCall.html#UG_GamsCall_SecondaryParameterFile).

**Comment by samgdotson** [comment]
Thanks! I didn't realize this about GAMS and sync'd folders. Though I'm not sure that explains the `Input line too long` error. Perhaps a comment about this would make sense for the troubleshooting documentation? It felt like I got lucky finding a workable solution so quickly.

**Comment by boxblox** [comment]
You are right, I suspect that this is not about your synch'd folder.  I only mention it because we always recommend users to run from a local directory if we notice that OneDrive/GoogleDrive is in the path.

**Comment by samgdotson** [comment]
Closing.
